configs/pet_wo_dog_and_cat.yaml

```diff
@@ -0,0 +1,16 @@
+cache_root: "/tmp/kandinsky2"
+output_dir: ""
+placeholder_token: "mypet"
+initializer_token: "pet"
+log_image_frequency: 250
+log_embed_frequency: 250
+img_size: 512
+train_batch_size: 1
+steps: 2500
+positive_classes: ["pet"]
+seed: 0
+live_negatives: True
+image_feature: True
+optimize_in_text_space: True
```

Comment:
the leaning process will focus on text space
such as the "mypet" (the new concept we want image generator to learn) will only encode to text space and train with positive and negative image space